src/Illuminate/Foundation/Testing/CrawlerTrait.php

```diff
@@ -93,6 +93,26 @@ public function post($uri, array $data = [], array $headers = [])
     }
 
     /**
+     * Visit the given URI with a POST request with content type of application/json.
+     *
+     * @param string $uri
```

Comment:
Sorry - what needs to be done? Got any links? I was just following the style in the rest of the file.